### PR TITLE
chore: add web utils

### PR DESCRIPTION
**Pull request by DaoDev44**
- Move some utils from `web` to `lib`

### CODE REVIEW

**Review by elmutt — DISMISSED** [review]

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]
One small request for type safety.

Also, you guys and your wild commit history 😅

**Comment by DaoDev44** [comment]
@0xApotheosis I know, the commit history is nuts. But it'll go away 😄

**Comment by 0xApotheosis** [comment]
> @0xApotheosis I know, the commit history is nuts. But it'll go away 😄

@DaoDev44 not mad, just impressed 🤯

**Review by 0xApotheosis — APPROVED** [review]

File: packages/caip/src/utils.test.ts [hunk 1]
```diff
@@ -72,3 +77,31 @@ describe('getChainReferenceFromChainId', () => {
...
   })
 })
+
+describe('chainIdToFeeAssetId', () => {
+  it('returns a chain fee assetId for a given ETH chainId', () => {
+    const result = chainIdToFeeAssetId(ethChainId)
+    expect(result).toEqual(ethAssetId)
+  })
+
+  it('returns chain fee assetId (ATOM) for a given Cosmos assetId', () => {
```
Comment (gomesalexandre) [inline packages/caip/src/utils.test.ts:87]:
```suggestion
  it('returns chain fee assetId (ATOM) for a given Cosmos chainId', () => {
```

File: packages/caip/src/utils.test.ts [hunk 2]
```diff
@@ -72,3 +77,31 @@ describe('getChainReferenceFromChainId', () => {
     expect(() => getChainReferenceFromChainId(invalidChainId)).toThrow()
   })
 })
+
+describe('chainIdToFeeAssetId', () => {
+  it('returns a chain fee assetId for a given ETH chainId', () => {
```
Comment (gomesalexandre) [inline packages/caip/src/utils.test.ts:82]: We're not very consistent with this across codebases, but let's try to keep the separate vocabulary for ETH the asset vs. Ethereum the network
```suggestion
  it('returns a chain fee assetId for a given Ethereum chainId', () => {
```

**Review by gomesalexandre — DISMISSED** [review]
Two small comments but LGTM! 🍬 

- Utils to be consumed in web are added and tested
- Comments are added noting that this is an intermediary implementation
- There is no vernacular reference in the added utils/tests

File: packages/caip/src/utils.test.ts (hunk 2 above)
Comment (DaoDev44) [reply packages/caip/src/utils.test.ts:82]: I like it. Good call

**Review by gomesalexandre — APPROVED** [review]
🍬

**Review by elmutt — APPROVED** [review]

**Review by 0xApotheosis — APPROVED** [review]

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**